pyproject.toml

```diff
@@ -0,0 +1,62 @@
+# pyproject.toml
+# See https://setuptools.pypa.io/en/latest/userguide/pyproject_config.html
+# for more information about the pyproject.toml file.
+
+# Recommended Packaging Guide
+# Run:
+# 1. `pip install --upgrade build` to install the build package.
+# 2. `python -m build` to build the package.
+# 3. `twine upload dist/*` to upload the package to PyPI.
+
+[build-system]
+requires = ["setuptools>=60"]
+build-backend = "setuptools.build_meta"
+
+[project]
+name = "truelearn"
+dynamic = ["version"]
+description = "A Machine learning library for recommending open educational resources"
+readme = "README.rst"
```

Comment:
We probably should use `README.md`?

Reply:
I was going to change it to readme.rst, to use that format since all the docs are generated in rst. I need to fix the syntax in #14

Reply:
> I was going to change it to readme.rst, to use that format since all the docs are generated in rst. I need to fix the syntax in #14 . The format markdown vs rst isn't very important, since we will create additional rst files for our docs to create a Getting started guide etc, using one format ensures consistency and will probably be easier to remember

I can probably fix the syntax in this PR

Reply:
Resolved